lib/block-editor-settings.php

```diff
@@ -74,6 +81,12 @@ function gutenberg_get_block_editor_settings( $settings ) {
 			$block_classes['css'] = $actual_css;
 			$global_styles[]      = $block_classes;
 		}
+		// Get any additional css from the customizer.
+		$global_styles[] = array(
+			'css'            => wp_get_custom_css(),
+			'__unstableType' => 'user',
+			'isGlobalStyles' => false,
+		);
```

Comment:
If this is going to be added anyway, would it make sense to do it above in `$global_styles = array();`? 

Or does order matter?

Reply:
The order matters for themes with theme.json: customizer CSS should be added before global styles custom CSS, so the latter can override the former.

Reply:
Got it, thanks for confirming 🙇🏻

Reply:
Actually, that wouldn't change if we moved it into the array declaration. The change would be moving customizer CSS before the general and block-level global styles. It's probably better to leave it as is because a hybrid theme that declares global styles may not provide access to the site editor, so the customizer may be the only option for adding custom CSS. And in that case, the custom CSS should be added after any global styles from theme.json.